notebooks/features/isolation_forest/IsolationForest - Multivariate Anomaly Detection.ipynb

```diff
@@ -644,7 +690,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install --upgrade raiwidgets interpret-community"
+    "# !pip install --upgrade raiwidgets interpret-community"
```

Comment:
Are these lines supposed to remain in the notebook, it seems they are commented out